domains/transfer/Transfer.API/API/appsettings.Development.json

```diff
@@ -21,6 +21,9 @@
     "TransferAgreementProposalCleanupService": {
         "SleepTime": "01:00:00"
     },
+    "TransferAgreementCleanup": {
+        "SleepTime": "01:00:00"
+    },
```

Comment:
So right now we have multiple workers starting every hour. will that be a potential problem now, or in the future?